draft-ietf-sipcore-sip-push.xml

```diff
@@ -38,68 +38,79 @@
-        service. Typically each operating system uses a dedicated push notification
-        service. For example, Apple iOS devices use the Apple Push Notification service (APNs).
+        In order to save resources (e.g., battery life) some devices (especially mobile devices)
+        and operating systems will suspend applications when they are not used. In some cases,
+        internal timers cannot be used to awake such application, nor will incoming network
+        awake the application. Instead, the only way to awake the application is by using a
+        push notification. Typically each operating system uses a dedicated Push Notification
+        Service (PNS). For example, Apple iOS devices use the Apple Push Notification service (APNs)
+        while Android devices use the Firebase Cloud Messaging (FCM) service.
       </t>
       <t>
-        Due to the restriction above, applications can not be awakened by non-push notification
-        traffic or internal timers. This means that a suspended SIP UA will not be able to receive 
-        an incoming SIP request (e.g., a SIP INVITE request), or to send periodic re-registration requests.
-        Also, the SIP UA will not be able to maintain connections e.g., using the SIP Outbound 
-        Mechanism <xref target="RFC5626"/>, as it requires the UA to send periodic keep-alive 
-        messages.
+        Because of the restrictions above, Session Initiation Protocol (SIP) User Agents (UAs)
+        <xref target="RFC3261"/> can not be awaken, in order to perform periodic re-registrations
+        and to process incoming calls, without using push notifications to awake the SIP UA
+        in order to perform those functions. 
       </t>
       <t>
-        This document describes how push notification mechanisms can be used to wake up
-        suspended SIP UAs, in order to be able to receive and generate SIP requests.
-        The document defines new SIP URI parameters, that can be used in a SIP REGISTER
-        request to provide push notification information from the SIP UA to the SIP entity
-        (realized as a SIP proxy in this document) that will send a push request to the 
-        push server in order to trigger a push notification towards the SIP UA.
-      </t> 
+        Also, without being able to use internal timers in order to awake applications, a SIP UA 
+        will not be able to maintain connections e.g., using the SIP Outbound Mechanism <xref target="RFC5626"/>, 
+        as it requires the UA to send periodic keep-alive messages.
+      </t>
+      <t>
+        This document describes how push notification mechanisms can be used to awake
+        suspended SIP UAs, in order to be able to perform re-registrations and to receive
+        incoming SIP requests for new calls. The document defines new SIP URI parameters that
+        to provide push notification information from a UA to the SIP entity (realized as a 
```

Comment:
defines new SIP URI parameters that to provide ->defines new SIP URI parameters that provide